debian/lighttpd/89-dump1090-fa.conf

```diff
@@ -21,26 +21,26 @@
 # server.modules += ( "mod_alias" )
 
 alias.url += (
-  "/dump1090-fa/data/" => "/run/dump1090-fa/",
-  "/dump1090-fa/" => "/usr/share/dump1090-fa/html/"
+  "/skyaware/data/" => "/run/",
```

Comment:
I am not a fan of this; it potentially exposes everything under `/run` to the outside world.

Can you alias e.g. `/skyaware/data-1090/` and `/skyaware/data-978/` separately insteal?